.github/workflows/ci.yml

```
@@ -174,7 +174,7 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@main
     with:
       sdist: false
-      test_extras: 'test'
+      test_extras: 'tests'
```

```suggestion
      test_extras: 'all,tests'
```
maybe this will work?

---

I don't think so, the idea here is not to install `all` because we don't want to pull in doc dependencies when testing.

---

all should not pull in docs. dev should.

---

Ah, I didn't realise, thanks!

---

It does mean we need to be more careful now with using `dev` as it can break CI or maybe fail to install on peoples machines.